bin/gobblin-mapreduce.sh

```diff
@@ -4,6 +4,9 @@
 ############### Run Gobblin Jobs on Hadoop MR ################
 ##############################################################
 
+# Set during the distribution build
+GOBBLIN_VERSION=@project.version@
```

Comment:
Thanks for your comments.
@sahilTakiar the idea is that _during the build when the tarball is created_ the 
```gobblin-mapreduce.sh``` is not just taken from bin/ but the project version is substituted as well. So the gobblin-mapreduce.sh in the tarball should contain something like this:
GOBBLIN_VERSION="0.6.0-233-g9f7dd61".